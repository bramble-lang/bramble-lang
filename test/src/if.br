fn my_main() {
    printiln if true {1} else {3};
    x:i32 := if 2 < 3 {4} else {5};
    printiln x;
    printiln (if (x < 1) {100} else {500});
    return;
}