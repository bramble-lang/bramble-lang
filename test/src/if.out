1
4
500
